mines/interaction_set[] trajectory_index
string frame_id
